docs/reference/sql/endpoints/jdbc.asciidoc

```diff
@@ -27,7 +27,19 @@ https://maven.apache.org/[Maven]-compatible tools can retrieve it automatically
 </dependency>
 ----
 
-from `artifacts.elastic.co/maven` by adding it to the repositories list:
+from https://search.maven.org/artifact/org.elasticsearch.plugin/x-pack-sql-jdbc[Maven Central Repository] by adding it to the repositories list:
+
+["source","xml",subs="attributes"]
+----
+<repositories>
+  <repository>
+    <id>central</id>
+    <url>https://repo.maven.apache.org/maven2</url>
+  </repository>
+</repositories>
+----
```

Comment:
Maven Central is already available as a repository in Gradle and Maven and as such, there's no need to add it. We should mention that it's available in mention central and point to the link and that's about it.
Only custom repos like our own are needed.

Reply:
Removed central repo example.